pages/dnd/reorderable-list.tsx

```diff
@@ -41,14 +40,14 @@ export function ReorderableList<Option extends { id: string }>({
       {staticOptions.map(option => (
         <li key={option.id}>{renderStaticOption?.(option)}</li>
       ))}
-      <DndContainer
+      <DndArea
         items={sortableOptions.map(option => ({ id: option.id, label: option.id, data: option }))}
         onItemsChange={items => onReorder([...staticOptions, ...items.map(item => item.data)])}
         renderItem={props => {
-          const className = clsx(props.isDragging && styles.placeholder, props.isSorting && styles.sorting);
+          const className = clsx(props.className, styles.option, props.isSorting && styles.sorting);
           const content = renderOption({ ...props, option: props.item.data });
           return props.isActive ? (
-            <Box>{content}</Box>
```

Comment:
Was the previous version not a `li` element because it is the element being dragged and therefore outside of the `ul`?

Reply:
That's a good point. I actually used the Box previously so that the font style is correct. The active node is rendered inside portal and does not have the font correctly inherited - I addressed that by adding `@include styles.default-text-style;` to the respective class name.

Ideally, I want the consumers to not depend on isActive and always return:
```
<li className={className} style={props.style}>
  {content}
</li>
```

That is possible for the list and container examples but not table where the column widths need to be set. I will dive deep to see if the semantics violation is a real problem here.

Reply:
Thanks for the explanation.

> I will dive deep to see if the semantics violation is a real problem here.

Were you able to take a look at this point?

Reply:
Yes, the semantic structure of the active node does not seem to have any effect. The active item is rendered in a portal and is detached from the actual list anyways - the screenreader users only receive the announcements really.

I propose that we use the simpler version to make the code better straightforward. Should we find a corner case with maybe some specific element we can change the integration slightly so that the active node is wrapped correctly - same as the active table row is wrapped with table and tbody now, yet it is done to ensure correct layout in that case.